CleverTapSDK/InApps/CustomTemplates/SystemAppFunction/CTSystemAppFunction.h

```diff
@@ -0,0 +1,22 @@
+//
+//  CTSystemAppFunction.h
+//  CleverTapSDK
+//
+//  Created by Nishant Kumar on 19/03/25.
+//  Copyright © 2025 CleverTap. All rights reserved.
+//
+
+#ifndef CTSystemAppFunction_h
+#define CTSystemAppFunction_h
+
+#import <Foundation/Foundation.h>
+#import "CTCustomTemplate.h"
+
+@interface CTSystemAppFunction : NSObject
+
++ (NSDictionary<NSString *, CTCustomTemplate *> *)getSystemAppFunctions;
```

Comment:
Rename to `systemAppFunctions` to be more objc.

Reply:
Done